bin/sqlline-thin.py

```diff
@@ -18,64 +18,58 @@
 # limitations under the License.
 #
 ############################################################################
-
+from __future__ import print_function
```

Comment:
I was curious what this was doing and it looks like it's some magic to make print in python3 work like it does in python2 (http://stackoverflow.com/questions/4560804/why-do-we-invoke-print-after-importing-print-function-in-python-2-6).